src/v3/src/components/Icon/CheckCircle.tsx

```diff
@@ -23,7 +23,6 @@ export const CheckCircle: FunctionComponent<IconProps> = ({ description }) => (
     xmlns="http://www.w3.org/2000/svg"
     role="img"
   >
-    <title>{description}</title>
     <path
```

Comment:
Not familiar with SVG `title`, does it not appear in the DOM so no snapshots need to be updated?

Reply:
Oh yea, snapshots will need to be updated :( committing that now